src/NuGet.Core/NuGet.ProjectModel/NuGet.ProjectModel.csproj

```diff
@@ -15,6 +15,10 @@
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="IsExternalInit" PrivateAssets="all" />
```

Comment:
In order for us to take a dependency on a new package within our NuGet.Core libraries, we need to ensure that it can be source built. Now this is a source only package so I'm guessing this is ok. 

@NikolaMilosavljevic @MichaelSimons 

Worth understanding if there any licensing disclosure considerations as well. 
I think we'd need to disclose it 3rd party notices, but I'm not sure this being a source packages affects that.

Reply:
We don't really need to use the package. It only adds a single file, which is trivial. I've done the equivalent thing in a subsequent commit and removed the package from the PR.